Javascript/exportimport.js

```diff
@@ -1,7 +1,14 @@
 function getRealTime(clock12h = false) {
```

Comment:
Don't need padding and it's much more concise.

```suggestion
const format24 = new Intl.DateTimeFormat("EN-GB", {
    year: "numeric",
    month: "2-digit",
    day: "2-digit",
    hour: "2-digit",
    hour12: false,
    minute: "2-digit",
    second: "2-digit"
})
const format12 = new Intl.DateTimeFormat("EN-GB", {
    year: "numeric",
    month: "2-digit",
    day: "2-digit",
    hour: "2-digit",
    hour12: true,
    minute: "2-digit",
    second: "2-digit"
})

function getRealTime(use12 = false) {
    format = use12 ? format12 : format24;
    let dateParts = format
        .formatToParts(new Date())
        .filter((x) => x.type != "literal")
        .reduce((a, x) => {a[x.type] = x.value; return a}, {});
    return `${dateParts.year}-${dateParts.month}-${dateParts.day} ${dateParts.hour}_${dateParts.minute}_${dateParts.second}${use12 ? "_" + dateParts.dayPeriod : ""}`
}
```